main.html

```diff
@@ -40,35 +42,120 @@
       </ul>
     </nav>
 
+
+     <header class="body" d-flex" style="overflow-x: hidden; overflow-y: hidden;">
+       <div class="overlay" style="position: absolute;"></div>
+     </header>
+
     <section class="home p-3 p-lg-5 d-flex d-column" id="home">
       <div class="my-auto">
         <h1 class="mb-0">Kelly Tang</h1>
-        <div class="subheading mb-5"> Electrical Engineering Student </div>
-        <p class="lead mb-5">I know how to solder hehe xd</p>
+        <div class="subheading mb-5"> University of Waterloo | Electrical Engineering Student </div>
+        <p class="lead mb-5">hello I know how to solder hehe xd</p>
       </div>
+
+ 
+
     </section>
 
     <section class="content p-3 p-lg-5 d-flex flex-column" id="about">
       <div>
         <h2 class="mb-5">About</h2>
-        Once upon a time, I woke up on time. im a loser
+        Hello there! I'm Kelly and I'm an engineering student at the University of Waterloo. My design experience ranges from PCB design to power distribution (grammar?). I'm interested in ???. Currently looking for job opportunities for fall 2019. pLS hire.

```

Comment:
bad change I like what it said before hehe. But this page is very empty we can work on it later